docs/management_api.yml

```diff
@@ -73,32 +73,18 @@ paths:
           format: integer
           default: 20
           maximum: 500
-        - name: created_before
```

Comment:
dropping content, old revision ?

Reply:
it's just that those changes appear this way because of indents - remove old block + insert as 'new' block